pythondev/slack.py

```diff
@@ -460,3 +462,30 @@ def add(self):
             message.text = message.text.format(number=len(members),
                                                user=event['user']['id'])
             await slack.send(message)
+
+    async def move_to(self, command, slack):
+        response = command.response()
```

Comment:
You can pass the argument `type_='ephemeral` here so you don't need to set it every time and code will be more readable.